### ISSUE TITLE
the peers.py and datatypes.py code is hard to follow

### ISSUE THREAD

**Issue by atramos** [skepticoin/skepticoin#72]
These two files contain multiple Python classes, making the code hard to follow and read.

Is there a special IDE plug-in that'll allow seeing all the classes on the navigation pane? If not, I'd like to refactor these files by breaking them up into individual file per class.

**Comment by lk16** [comment]
I'll go ahead and work on splitting peer.py.

**Comment by atramos** [comment]
I think purging invalid peers is probably more urgent. We have less than 5 peers online and the file keeps growing.

**Comment by atramos** [comment]
Closing this for now, I'm ok with using VSCode "OUTLINE" view.
